src/yadism/observable_name.py

```diff
@@ -50,6 +50,13 @@ def name(self):
         """joint name"""
         return self.kind + "_" + self.flavor
 
+    @property
+    def is_parity_violating(self):
```

Comment:
Not a strong opinion, but for these names I usually prefer dropping the `is_` for simplicity.
On one side it is more or less the same, on the other it is slightly more consistent, because it is a property (the `is_something()` is more consistent for a function, since it is an action, but a property is supposed to be a pseudo-attribute, so it should pretend to be an entity, not an action).

Reply:
This, I do not also have a strong opinion, but at least doing it this way is consistent with what are currently there:

https://github.com/NNPDF/yadism/blob/0e0dc11ad9e959ba17f380ace38d21ff28187fd3/src/yadism/observable_name.py#L113-L114

https://github.com/NNPDF/yadism/blob/0e0dc11ad9e959ba17f380ace38d21ff28187fd3/src/yadism/observable_name.py#L125-L126

https://github.com/NNPDF/yadism/blob/0e0dc11ad9e959ba17f380ace38d21ff28187fd3/src/yadism/observable_name.py#L130-L131

https://github.com/NNPDF/yadism/blob/0e0dc11ad9e959ba17f380ace38d21ff28187fd3/src/yadism/observable_name.py#L135-L136

https://github.com/NNPDF/yadism/blob/0e0dc11ad9e959ba17f380ace38d21ff28187fd3/src/yadism/observable_name.py#L140-L141

So, in order to be consistent everywhere we'd also need to replace the rests, which probably should better be done in a different PR.